test/react-native/features/fixtures/rn0.60/ios/reactnative.xcodeproj/project.pbxproj

```diff
@@ -17,11 +17,16 @@
 		2D02E4BD1E0B4A84006451C7 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 13B07FB51A68108700A75B9A /* Images.xcassets */; };
 		2D02E4BF1E0B4AB3006451C7 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
 		2DCD954D1E0B4F2C00145EB5 /* reactnativeTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* reactnativeTests.m */; };
+		5F609DB53156E4476F05E7AF /* libPods-reactnativeTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 068E93DE6A5D0A0DF22ACF41 /* libPods-reactnativeTests.a */; };
```

Comment:
Presumably analogous changes will be needed to the RN 0.63 project once the branches meet, but we're unlikely to forget to do so as the tests will fail when trying to run the scenarios?